Gruntfile.js

```diff
@@ -106,12 +106,11 @@ module.exports = function (grunt) {
         },
         lintspaces: {
             src: [
-                "*",
-                "src/**/*",
-                "test/**/*",
-                "!**/*.ogg",
-                "!**/*.otf",
-                "!**/*.png"
+                "*.js",
```

Comment:
I don't like the idea of not checking the svg files as well as the other text files in the root directory (the README, LICENSE, Gruntfile, etc.). How much time does this check actually save? More than 1s?